### DISCUSSION TITLE
[Flow Manager] - suppress commands logs

### DISCUSSION THREAD

**Discussion by vguccione01** [mia-platform/community#407] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

1.96.3

### Description

Good morning,
I am using the Flow Manager saga pattern in different projects. I have many node microservices that interact with Flow Manager through the Flow Manager Client (https://github.com/mia-platform/flow-manager-client).
I'm wondering if is possible to suppress the following log that appear when a microservice do not implement an handler for a specific command, for example when the command is handled by another microservice, to have a more readable log stream

```
{"level":"WARN","timestamp":"...","name":"...","sagaId":"...","topic":"...","partition":4,"offset":"12","command":"...","message":"command not executed because is it not expected"}
```

### Actual Outcome

```
{"level":"WARN","timestamp":"...","name":"...","sagaId":"...","topic":"...","partition":4,"offset":"12","command":"...","message":"command not executed because is it not expected"}
```

### Expected Outcome

Some kind of configuration to suppress the log `command not executed because is it not expected` and have a more readable log stream

**Comment by fredmaggiowski** [comment]
Hi @vguccione01 thanks for reaching out,

I think your proposal makes totally sense, if you agree we can transfer this discussion to the [Flow Manager client](https://github.com/mia-platform/flow-manager-client) issues so that it can properly be addressed there.

**Reply by vguccione01** [reply]
Of course, no problems for me! Thank you!

**Reply by fredmaggiowski** [reply]
If you don't mind, I'll let you open the issue yourself so that it remains assigned to you :)

**Reply by vguccione01** [reply, marked as answer]
Sure!
https://github.com/mia-platform/flow-manager-client/issues/159

**Reply by fredmaggiowski** [reply]
Thanks!
